README.md

```diff
@@ -134,17 +134,49 @@ make a call to the `get` method by running:
 yab -t ~/keyvalue.thrift -p localhost:12345 keyvalue KeyValue::get -r '{"key": "hello"}'
 ```
 
-This specifies a single `host:port` using `-p`, but you can also specify multiple peers
+#### Proto
+The following examples assume that the Proto service running looks like:
+```proto
+service KeyValue {
+  rpc GetValue(Request) returns (Response) {}  
+}
```

Comment:
You should also add the definition of `Request` and `Response` since there are not a well known type (this differs from Thrift since you can not use string here)